src/screens/Library/components/InstallModal/index.css

```diff
@@ -19,6 +19,7 @@
 }
 
 .modal {
+  position: relative !important;
```

Comment:
Is the `!important` needed here? can you try without it?

Reply:
I removed it and it works fine 👍